### PR TITLE
Typo "*none of defaultMail@contoso.com*"→"*none of defaultMail\@conto…

### PR DESCRIPTION
**Pull request by hyoshioka0128**
…so.com*"

Italic with escape characters

https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered?view=o365-worldwide

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Threat Explorer is a powerful report that can serve multiple purposes, such as f
...
 
-   Advanced filtering is a great addition to search capabilities. A boolean **NOT** filter has been introduced on *Recipient*, *Sender* and *Sender domain* to allow admins to investigate by excluding values. This option appears under selection parameter *Contains none of*. **NOT** will let admins exclude alert mailboxes, default reply mailboxes from their investigations, and is useful for cases where admins search for a specific subject (subject="Attention") where the Recipient can be set to *none of defaultMail@contoso.com*. This is an exact value search.
+   Advanced filtering is a great addition to search capabilities. A boolean **NOT** filter has been introduced on *Recipient*, *Sender* and *Sender domain* to allow admins to investigate by excluding values. This option appears under selection parameter *Contains none of*. **NOT** will let admins exclude alert mailboxes, default reply mailboxes from their investigations, and is useful for cases where admins search for a specific subject (subject="Attention") where the Recipient can be set to *none of defaultMail\@contoso.com*. This is an exact value search.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md:86]:
```suggestion
   Advanced filtering is a great addition to search capabilities. A boolean **NOT** filter has been introduced on *Recipient*, *Sender*, and *Sender domain* to allow admins to investigate by excluding values. This option appears under selection parameter *Contains none of*. **NOT** will let admins exclude alert mailboxes, default reply mailboxes from their investigations, and is useful for cases where admins search for a specific subject (subject="Attention") where the Recipient can be set to *none of defaultMail\@contoso.com*. This is an exact value search.
```

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md:86]: Please add a comma after "*Recipient*".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit.

**Pull request merged**